frontend/support/data/builds.js

```diff
@@ -0,0 +1,1349 @@
+export const builds = [
```

Comment:
Eventually we should create some standardized way of creating our fixtures but this is perfect for now